src/test/suite/asciidocUtilsTest.test.ts

```diff
@@ -42,4 +49,18 @@ suite('AsciiDoc Utils Test Suite', () => {
 		expect(content).to.include('The quick brown fox jumps over the lazy dog.');
 	});
 
+	test('get the adoc parser', () => {
+		expect(adocutils.getADParser()).to.not.be.null;
+	});
+
+	test('that we can parse adoc to html', () => {
+		const htmlOutput = adocutils.parseADtoHTML(BASIC_EXAMPLE);
+		expect(htmlOutput).to.include(`<h1>Document Title</h1>`);
+		expect(htmlOutput).to.include(`<p>That&#8217;s all, folks!</p>`);
+
+		// this is tested more effectively with 'open an asciidoc file with an include' above
```

Comment:
if tested more effectively by the other test. What is the specific purpose of this one?

Reply:
the use of the method is buried and not immediately apparent to anyone actually looking at the utility class itself. this serves as a simple test that doubles up on what is already done in the case we test above with opening an adoc with an include as a didact adoc

Reply:
somewhat duplicated earlier test to show the utility method in action and split it into separate test